app/views/events/_form.html.erb

```diff
@@ -15,77 +15,103 @@
       <%= f.text_area :description, id: :event_description, class: 'form-control' %>
     </div>
   </div>
+
   <div class="form-group">
-    <%= f.label :event_type, class: 'control-label col-lg-2' %>
+    <%= f.label :event_type, :class => 'control-label col-lg-2' %>
```

Comment:
We decided to use the colon notation in favor of the arrow notation for mapping key value pairs in hashes.